styles.css

```diff
@@ -266,8 +261,56 @@ input {
 
 
 /* --------------------------------------------------->> */
-
 /*** LOG SECTION ***/
+.card-container {
+  text-align: left;
+  height: 100px;
+  background: #46424D;
+  border-radius: 5px;
+  color: white;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  margin: 10px 0;
+}
+
+.card-header {
+  width: 95%;
+  margin: 10px 15px;
+
+}
+.sliver {
+  height: 30px;
+  border-radius: 2px;
+  margin: 10px 10px 0 0;
+}
+
+.sliver-purple {
+  background: #C278FD;
+  border-right: 2px solid #C278FD;
+  border-left: 2px solid #C278FD;
+}
+
+.sliver-green {
+  background: #B3FD78;
+  border-right: 2px solid #B3FD78;
+  border-left: 2px solid #B3FD78;
+}
+
+.sliver-red {
+  background: #FD8078;
+  border-right: 2px solid #FD8078;
+  border-left: 2px solid #FD8078;
+}
+
+.card-category {
+  font-weight: bold;
+  color: white;
+}
+
+.card-body {
+  margin: 10px 5px;
+}
 
```

Comment:
Nice work getting these cards formatted - they look identical to comp. Only issue that I'm seeing is the card is displaying an activity time of 0:00, rather than the original time that was submitted.